apps/dotcom/client/src/tla/app/zero-polyfill.ts

```diff
@@ -203,6 +203,8 @@ export class Zero {
 		console.log('apply local optimistic updates', mutationId, updates)
 		this.store.updateOptimisticData(updates, mutationId)
 
+		if (this.socket.isDisposed) return
```

Comment:
Looks like the error is because:
- We first [dispose](https://github.com/tldraw/tldraw/blob/f716f149ff5c82ac9a901a46ce43ede2d9071d9b/apps/dotcom/client/src/tla/hooks/useAppState.tsx#L55) the app.
- Only then we call [onFileExit](https://github.com/tldraw/tldraw/blob/f716f149ff5c82ac9a901a46ce43ede2d9071d9b/apps/dotcom/client/src/tla/components/TlaEditor/TlaEditor.tsx#L200) from the `TlaEditor`